repo2docker/app.py

```diff
@@ -11,15 +11,15 @@
 import sys
 import logging
 import os
+import entrypoints
```

Comment:
This is a new dependency, but it's already used in other Jupyter projects such as jupyterhub:
https://github.com/jupyterhub/jupyterhub/blob/bc7bb5076ff2deabc7702c75dce208166d14568e/requirements.txt#L4